fairroot.sh

```diff
@@ -1,6 +1,6 @@
 package: FairRoot
 version: "%(short_hash)s"
-tag: "6af7e848585ce720e96a01d5565b1bd5e3884430"
+tag: "4ab2d2b1f0392ad93b92026d70fe049c38665f7d"
```

Comment:
The FLP people cannot use hashes for the version, because that does not work with their Updateable RPMS paradigm. Can you please tag this?

Reply:
There is no tag including this development. The version currently used is also specified by a hash, so thy have already to deal with a hash now. But we can, of course, ask FairRoot developers for a tag.

Reply:
I will let @awegrzyn @Barthelemy @vascobarroso to comment, but my understanding is that the updateable RPM paradigm the need, requires properly tagged versions.

Reply:
Indeed, as far as I know the hashes are not ok for the RPMs. They confuse the system as it does not know which version is more recent. If there is already a tag, indeed, we are already in a bad place. I would therefore propose the following (if @vascobarroso and @awegrzyn ) agree : keep the hash here to go ahead but in parallel could you ask FairRoot to tag and prepare subsequently a PR with a proper version ?

Reply:
@Barthelemy , I asked for a tag yesterday in my PR in FairRoot (this PR includes the update in CMake which we need for the developments in this PR, and which is included in the new hash in this PR). When a tag is available, we will move to it in a new PR.